packages/shared/components/Select/types.ts

```diff
@@ -37,6 +37,9 @@ export type Props = {
   components?: any;
   customProps?: Record<string, any>;
   menuPosition?: 'fixed' | 'absolute';
+  inputValue?: string;
+  filterOption?(): null | boolean;
+  onInputChange?(value: string, actionMeta: { action: InputAction }): void;
```

Comment:
Maybe extract type of this function to the new type (so we will be able to use it in handler in `SelectFilters.tsx` line 84)?